### DISCUSSION TITLE
Is there an OpenFlow project suitable for OMNet++ 6.0.3 and INET framework 4.5.2?

### DISCUSSION THREAD

**Discussion by LeoChristopherHub102** [omnetpp/omnetpp#1186] [Q&A]
I want to simulate an MPLS network that uses SDN (Software Defined Networking) controller for dynamic management of network policies and the IDS/IPS (Intrusion Detection System/Intrusion Prevention System) strategy points.

Is an OpenFlow project published for OMNet++ 6.0.3 and INET framework 4.5.2?

**Comment by rhornig** [comment]
No. It was not ported to a newer version of INET / OMNeT++ AFAIK.

**Reply by LeoChristopherHub102** [reply]
Thx.
Then, Which versions of OMNet++ and INET framework environments can the current OpenFlow run on?

**Reply by adamgeorge309** [reply]
Works with OMNeT++ 5.5.1 and INET 3.6.6

**Comment by avarga** [comment]
Try 'opp_env install openflow-latest'.
opp_env: https://github.com/omnetpp/opp_env/tree/main/opp_env

**Comment by aarizaq** [comment]
You can try this
https://github.com/aarizaq/openflow/tree/openFlow-inet4.3. It shold work with inet 4.5.2. If you have a problem you can report to me
